perun-core/src/main/resources/perun-core.xml

```diff
@@ -270,6 +270,25 @@ http://www.springframework.org/schema/aop http://www.springframework.org/schema/
 		<constructor-arg ref="perun" />
 	</bean>
 
+    <bean id="extSourceCSV" class="cz.metacentrum.perun.core.impl.ExtSourceCSV">
```

Comment:
Hi, Spring somehow doesn't like it on run-time causing null pointer exception. Please change all new extSource beans like this:
```xml
<bean class="cz.metacentrum.perun.core.impl.ExtSourceLdap" factory-method="setPerunBlImpl">
	<constructor-arg ref="perun" />
</bean>
```
It's a little hack, which makes spring call static method. Notice removed *id* which is not necessary if bean is not used anywhere else for inclusion. Also code part in each ExtSource class needs to be changed to this:

```java
private static PerunBlImpl perunBl;

// filled by spring (perun-core.xml)
public static PerunBlImpl setPerunBlImpl(PerunBlImpl perun) {
	perunBl = perun;
	return perun;
}
```
Thank you.